tibanna_ffcommon/generic_qc_utils.py

```diff
@@ -1,7 +1,160 @@
-from .exceptions import (
-    GenericQcException
-)
-from typing import Any, List
+from .exceptions import GenericQcException
+from typing import Any, List, Union, Optional
+from pydantic import BaseModel, ConfigDict, ValidationError, RootModel
+from .misc_utils import LogicalExpressionParser
+
+# Tibanna interal QC flags
+PASS = "pass"
+WARN = "warn"
+FAIL = "fail"
```

Comment:
You might want to import and reference these ie: in unit tests where these strings are hard coded